ext/storage_sources/azure_store/README.md

```diff
@@ -0,0 +1,106 @@
+Check your g++ version typing `g++ --version`.
+
+### How to install requirements (skip this step if requirements have been met)
+
+If the CMake version is not 3.13 or higher update your CMake to 3.13 using the following.
+```bash
+sudo apt remove cmake
+wget https://cmake.org/files/v3.13/cmake-3.13.0.tar.gz
+tar xf cmake-3.13.0.tar.gz
+
+cd cmake-3.13.0
+
+./configure
+make -j $(nproc)
+sudo make install
+```
+
+Check that your CMake has been updated using the following command `cmake --version`.
+
+If your cmake is not in `/usr/bin/` create a symbolic link using the following.
+```bash
+sudo ln -s /usr/local/bin/cmake /usr/bin/cmake
+```
+
+If the G++ version is not 8.4 or higher update your G++ to 8.4 using the following.
+```bash
+sudo apt-get install gcc-8 g++-8
+sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 20 --slave /usr/bin/g++ g++ /usr/bin/g++-8
+```
+
+Check that your G++ has been updated using the following command `g++ --version`.
+
+### Building
+
+There is currently only 1 way to build WiredTiger with Azure extension:
+1. Letting CMake manage the Azure SDK dependency as an external project, letting it download, link and build the extension.
+
+There are two CMake flags associated with the Azure extension: `ENABLE_AZURE` and `IMPORT_AZURE_SDK`.
+* `ENABLE_AZURE=1` is required to build the Azure extension.
+* `IMPORT_AZURE_SDK={external}` is used to set the build method.
+    *   `external` tells the compiler to search for an existing system installation of the SDK.
+    *    This flag should be set alongside the `ENABLE_AZURE` flag.
+    *    If the `IMPORT_AZURE_SDK` flag is not specified, the compiler will assume a system installation of the SDK which will be implemented at a later stage.
+
+### Letting CMake manage the SDK dependency as an external project
+
+This method configures CMake to download, compile, and install the Azure SDK while building the Azure extension.
+
+```bash
+# Create a new directory to run your build from
+mkdir build && cd build
+
+# Configure and run cmake with Ninja
+cmake -DENABLE_PYTHON=1 -DHAVE_UNITTEST=1 -DIMPORT_AZURE_SDK=external -DENABLE_AZURE=1 -G Ninja ../.
+ninja
+```
+
+* The compiler flag `IMPORT_AZURE_SDK` must be set to `external` for this build method.
+* `ENABLE_AZURE` defaults to looking for a local version, the `IMPORT_AZURE_SDK` setting will override that default.
+## 3. Development
+In order to run this extension after building, the developer must have an Azure connection string locally to a container with the right permissions. The connection string must be stored in an environmental variable called `AZURE_STORAGE_CONNECTION_STRING`. To store your connection string into an environmental variable type `export AZURE_STORAGE_CONNECTION_STRING="your Azure connection string"` into your terminal.
+## 4. Testing
+
+Before running the tests set the `LD_LIBRARY_PATH` to tell the dynamic link loader where to look for the dynamic shared libraries that we made earlier.
```

Comment:
It's conventional to just say "the linker" rather than "the dynamic linker/loader" :-)

Reply:
fixed in https://github.com/wiredtiger/wiredtiger/pull/8802/commits/7a0835b2f48fa7ee3afc558eac783f798980748f